exporters/metric/cortex/example/config.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+url: http://host.docker.internal:9009/api/prom/push
+remote_timeout: 30s
+push_interval: 2s
+name: Test
+headers:
+  X-Scope-OrgID: 5
```

Comment:
I take it that this X-Scope-OrgID value is necessary and arbitrary, here?

Reply:
Yes, it's required in the exporter config but ignored by the "test" Cortex config we're running.